pkg/local_object_storage/blobovnicza/delete.go

```diff
@@ -29,12 +31,18 @@ func (p *DeletePrm) SetAddress(addr oid.Address) {
 // Returns an error of type apistatus.ObjectNotFound if the object to be deleted is not in blobovnicza.
 //
 // Should not be called in read-only configuration.
-func (b *Blobovnicza) Delete(prm DeletePrm) (DeleteRes, error) {
+func (b *Blobovnicza) Delete(prm DeletePrm) (res DeleteRes, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = fmt.Errorf("%w: %v", ErrFatal, err)
```

Comment:
Does `err != nil` always in case of panic?

Reply:
Actually it is always nil, should be `r`, fixed.